website/docs/d/is_ike_policies.html.markdown

```diff
@@ -43,7 +43,7 @@ In addition to all argument references listed, you can access the following attr
 	- `key_lifetime` - (Integer) The key lifetime in seconds.
 	- `name` - (String) The user-defined name for this IKE policy.
 	- `negotiation_mode` - (String) The IKE negotiation mode. Only `main` is supported.
-	- `resource_group` - (List) The resource group for this IKE policy.
+	- `resource_group` - (List) The resource group, for this IKE policy.
```

Comment:
can we have same consistent 
```
`resource_group` - (List) The resource group object,....
```

Reply:
corrected from 
`resource_group - (List) The resource group,`  
 to 
`resource_group - (List) The resource group object,....`
across multiple files.